docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+tutorials.
+
+</div>
+
+* Table of Contents
+    * **[Quick Start](#quick-start)**
+    * **[Features](#features)**
+        * **[Student](#students)**
+            * Adding a student: [`addstu`](#adding-a-student--addstu)
+            * Editing a student: [`editstu`](#editing-a-student-editstu)
+            * Adding student's attendance: [`attendance`](#adding-students-attendance--attendance)
+            * Adding student’s response: [`addresponse`](#adding-students-response-addresponse)
+            * Adding help tag to a student: [`helpstu`](#adding-help-tag-helpstu)
+            * Deleting a student: [`deletestu`](#deleting-a-student-deletestu)
+            * Finding a student: [`findstu`](#finding-a-student-findstu)
+            * Listing all students: [`liststu`](#listing-all-students-liststu)
+        * **[Questions](#questions)**
+            * Adding a question: [`addq`](#adding-a-question--addq)
+            * Marking a question: [`markq`](#marking-a-question--markq)
+            * Unmarking a question: [`unmarkq`](#unmarking-a-question--unmarkq)
+            * Deleting a question: [`deleteq`](#deleting-a-question--deleteq)
+        * **[Tutorials](#tutorials)**
+            * Adding a tutorial: [`addtut`](#adding-a-tutorial--addtut)
+            * Deleting a tutorial: [`deletetut`](#deleting-a-tutorial--deletetut)
+            * Marking a tutorial: [`marktut`](#marking-a-tutorial-marktut)
+        * **Exiting the program**: [`exit`](#exiting-the-program--exit)
+    * **[FAQ](#faq)**
+    * **[Command Summary](#command-summary)**
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Quick Start
+
+1. Ensure you have Java `11` or above installed in your Computer.
+
+1. Download the latest `SETA.jar` from [here](https://github.com/AY2223S1-CS2103T-T08-4/tp/releases).
+
+1. Copy the file to the folder you want to use as the _home folder_ for your SETA.
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app
+   contains some sample data.<br>
+   ![Ui](images/Ui.png)
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`liststu`** and pressing Enter will
+   list all the students in the 'Student' column.<br>
+
+Some example commands you can try:
+
+* **`addstu`**`n/John Lim Jun Jie h/@johnlimjj e/johnlim@example.com` : Adds a student named `John Lim Jun Jie` to the
+  student list.
+
+* **`deletestu`**`3` : Deletes the 3rd student shown in the current list.
+
+* **`exit`** : Exits the app.
+
+Refer to the [Features](#features) below for details of each command.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Features
+
+SETA contains features that allow you to manage 3 kinds of lists:
```

Comment:
Phrasing here could be 3 things instead of 3 kinds of lists? 😓

Reply:
done